README.rst

```diff
@@ -0,0 +1,148 @@
+Nameko eventlog dispatcher
+==========================
+
+.. pull-quote::
+
+    `Nameko <http://nameko.readthedocs.org>`_ dependency provider that
+    dispatches log data using ``Events`` (Pub-Sub).
+
+
+Usage
+-----
+
+Dispatching event log data
+``````````````````````````
+
+Include the ``EventLogDispatcher`` dependency in your service class:
+
+.. code-block:: python
+
+    from nameko.rpc import rpc
+    from nameko_eventlog_dispatcher import EventLogDispatcher
+
+
+    class FooService:
+
+        name = 'foo'
+
+        eventlog_dispatcher = EventLogDispatcher()
+
+        @rpc
+        def foo_method(self):
+            self.eventlog_dispatcher('foo_event_type', {'value': 1})
+
+``event_type`` and  some ``event_data`` (optional) will be provided as
+arguments. ``event_data`` must contain serializable data.
+
+Calling ``foo_method`` will dispatch an event from the ``foo`` service
+with ``log_event`` as the event type.
+
+Then, any `nameko <http://nameko.readthedocs.org>`_ service will be able
+to handle this event.
+
+.. code-block:: python
+
+    from nameko.events import event_handler
+
+
+    class BarService:
+
+        name = 'bar'
+
+        @event_handler('foo', 'log_event')
+        def foo_log_event_handler(self, body):
+            """`body` will contain the event log data."""
+
+
+Capturing log data when entrypoints are fired
+`````````````````````````````````````````````
+
+Enable auto capture event logs in your
+`nameko <http://nameko.readthedocs.org>`_ configuration file:
+
+.. code-block:: yaml
+
+    # config.yaml
+
+    EVENTLOG_DISPATCHER:
+      auto_capture: true
+      entrypoints_to_exclude: []
+
+Then some events will be atomatically captured and dispatched as
+`nameko <http://nameko.readthedocs.org>`_ Events. For instance, when an
+entrypoint is fired.
+
+There are some things to take into consideration:
+
+- This will be enabled by setting ``auto_capture`` to ``true``.
+- The source service for these events will be ``all``.
```

Comment:
As we discussed. I'm going to remove ``all`` as a source service in my next PR. Some reasons:
- Simplifies the dependency provider.
- It will be less coupled to the Nameko internals.
- All the event log data will be captured per service and using a single event type, which looks like a better design at the moment.

Thanks for the comment.